keyboards/dz60/keymaps/via/rules.mk

```diff
@@ -0,0 +1,3 @@
+MOUSEKEY_ENABLE = no
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```

Comment:
Does enabling VIA without the Mouse Keys and LTO changes make the firmware too large to flash when compiled?

Reply:
> The Org60 and remote tracking merges prompted me to look into the state of your fork.
> 
> If you follow the instructions in the following docs, you should be able to sync your fork more easily:
> 
> * [Resynchronizing a Branch - QMK Firmware Docs](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch)
> * [Using Your Fork's Master - QMK Firmware Docs](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)
> 
> If you need any help with this, let me know.

Thanks, I noticed that way too late. Should I recreate a PR with a fresh state? I can't seem to fix that at this point without recreating

Reply:
Those rules are VIA keymap specific, so it should. I've also verified locally that there are same build results with and without them.